src/interface/src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
```

Comment:
Actually, should we wrap legend into its own module, so we just import what we need instead of the whole shared module?

Reply:
Hmm, I assumed that we would put more custom components into the shared module, and then importing one module (shared) instead of several (one for each component) into AppModule would be cleaner.

Reply:
Yeah that makes sense, but I'm wondering if legend would be shared by multiple features/components? We can have it  in shared for now and see where it goes.